lib/publiccloud/provider.pm

```diff
@@ -41,10 +41,34 @@ sub init {
     my ($self) = @_;
     my $file = lc get_var('PUBLIC_CLOUD_PROVIDER');
     assert_script_run('mkdir -p ' . TERRAFORM_DIR);
-    assert_script_run('curl ' . data_url('publiccloud/terraform/' . $file . '.tf') . ' -o ' . TERRAFORM_DIR . '/plan.tf');
+    if (check_var('SLES4SAP_MODE', 'publiccloud')) {
```

Comment:
what is correlation between `PUBLIC_CLOUD_SLES4SAP` and `SLES4SAP_MODE` ?

Reply:
In fact I think that `SLES4SAP_MODE` is not needed anymore. I'm testing with `PUBLIC_CLOUD_SLES4SAP` only.

Reply:
Yes, `SLES4SAP_MODE` is not needed anymore, so I removed it.